rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1062,152 +542,20 @@ void MasterGraph::output_routine()
             _ring_buffer.set_meta_data(full_batch_image_names, full_batch_meta_data);
             _ring_buffer.push(); // Image data and metadata is now stored in output the ring_buffer, increases it's level by 1
         }
-        _process_time.end();
-
-    }
-    catch (const std::exception &e)
-    {
-        ERR("Exception thrown in the process routine: " + STR(e.what()) + STR("\n"));
-        _processing = false;
-        _ring_buffer.release_all_blocked_calls();
```

Comment:
why is this exception handling code removed?

Reply:
We have removed the output routine of the video, so it is showing this as removed but the code is present for the same